### ISSUE TITLE
ng-repeat inside template does not seem to work (demo)

### ISSUE THREAD

**Issue by pandaiolo** [akoenig/angular-deckgrid#43]
Why is it not possible to use `ng-repeat` inside a `deckgrid` while `ng-repeat` can be used recursively ?

Demonstration of both non-working (deckgrid) and working (ng-repeat) behaviour here : (http://codepen.io/anon/pen/nfzHk/)

**Comment by zenithtekla** [comment]
Indeed, it does work. I was able to have ng-repeat div work within the template file and render different templates for different listing styles. I have experience and confidence working with Directives in AngularJS.
You are welcome to check out my repo for solution to your end: [my repo - R37](https://github.com/zenithtekla/asmx-data-retrieval/commit/217566b296ebd3c34a3bb938e998eca9432c3394)
